sandbox/create_admin.py

```diff
@@ -1,20 +1,26 @@
+import hashlib
+
 from redisConnections.redis_users_connection import RedisUsersConnection
 from utility.create_config import create_config
 
 
+def hash_pw(password: str) -> str:
```

Comment:
we have this function https://github.com/YangCatalog/backend/blob/develop/api/authentication/auth.py#L17 which hashes the password before it gets checked, so I believe we shouldn't have two functions for hashing the password because if one changes then another one should be changed to, maybe it would be better to place this function somewhere else and use it in this script and in auth.py as well?